tests/test_safety.py

```diff
@@ -127,7 +127,7 @@ def test_review_from_file(self):
             input_vulns = read_vulnerabilities(insecure)
 
         vulns = safety.review(input_vulns)
-        assert(len(vulns), 3)
```

Comment:
unrelated bug - the assert was just checking that `len(vulns)` > 0, rather than 3